.gitignore

```diff
@@ -11,6 +11,7 @@ release.properties
 **/.prefs
 **/.classpath
 /target/
+**/bin/*
```

Comment:
Ok, I'm going to remove it.

Reply:
I have no idea why would VSCode create anything there - IntelliJ does not create there anything. But the folder contains the startup scripts. So it cannot be ignored.

Reply:
I guess you either need to ignore the specific files and change VSCode to not add any files there.

Reply:
Ok, this is not a problem at all, but in these bin directories I see a mix of source and target files. Shouldn't we have something like the following in the gitignore, rather than having to create a manual configuration at the editor level?
```
# Generated assembly files
**/bin/**/*.class
**/bin/**/*.yaml
**/bin/**/*.out
**/bin/**/*.out
api-conversion/bin/src/*
api-conversion/bin/Makefile
api-conversion/bin/pom.xml
api-conversion/bin/README.md
```

Reply:
TBH, I think you need to figure out what adds the file there. There is nothing like that in this directory in the master branch: https://github.com/strimzi/strimzi-kafka-operator/tree/master/api-conversion/bin and there is nothing like that in the bin directory in any of my working clones. The content you seem to have there suggests something just copied stuff there by mistake / error. So in general, the better way would be to make sure it is not there in the first place.

Reply:
Ok, I will revert that chaneg as it is unrelated to this PR. Anyway, I'm just using maven to do the project build, so it must be the output of some plugin.